### ISSUE TITLE
feat: Kubernetes multiple worker groups

### ISSUE THREAD

**Issue by lukasmrtvy** [apache/cloudstack#5756]
##### ISSUE TYPE

 * Feature Idea
##### COMPONENT NAME

~~~
Kubernetes
~~~
##### IDEA
Would be superb to implement support for multiple worker groups in Kubernetes component ( major clouds GCP,AWS,Azure and Openstack are supporting it ). Thanks

**Comment by nvazquez** [comment]
Thanks @lukasmrtvy for your proposal cc. @rohityadavcloud @davidjumani @Pearl1594

**Comment by DaanHoogland** [comment]
@davidjumani is this feasible (for 4.18, i.e. the next two weeks)?

**Comment by davidjumani** [comment]
@DaanHoogland Nope, but a CAPC cluster with multiple worker groups can easily be created if the cluster spec yaml file is configured to do so

**Comment by shwstppr** [comment]
We do not have any work in progress for this at the moment so moving it out of 4.19.0.0 milestone
